src/main/java/mate.academy/lib/Injector.java

```diff
@@ -1,13 +1,77 @@
 package mate.academy.lib;
 
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Field;
+import java.util.HashMap;
+import java.util.Map;
+import mate.academy.service.FileReaderService;
+import mate.academy.service.ProductParser;
+import mate.academy.service.ProductService;
+import mate.academy.service.impl.FileReaderServiceImpl;
+import mate.academy.service.impl.ProductParserImpl;
+import mate.academy.service.impl.ProductServiceImpl;
+
 public class Injector {
     private static final Injector injector = new Injector();
+    private static final Map<Class<?>, Class<?>> interfacesImplementations = new HashMap<>();
+    private final Map<Class<?>, Object> instances = new HashMap<>();
+
+    static {
+        interfacesImplementations.put(FileReaderService.class, FileReaderServiceImpl.class);
+        interfacesImplementations.put(ProductParser.class, ProductParserImpl.class);
+        interfacesImplementations.put(ProductService.class, ProductServiceImpl.class);
+    }
```

Comment:
can be initialized right in the field as follows: `interfacesImplementations = Map.of(key, value, key, value, ...)`